packages/block-editor/src/components/warning/style.scss

```diff
@@ -44,11 +45,16 @@
 
 	.block-editor-warning__action {
 		margin: 0 6px 0 0;
+
+		.components-button {
```

Comment:
>The buttons are poorly aligned because components-button.is-default has vertical-align: top while components-button.is-primary doesn't

Why not add this to `components-button.is-primary`?

Reply:
I think the right way to go would be to remove `vertical-align: top` from the `components-button.is-default`, as I can't see why it's needed—the default of `baseline` makes more sense.

It's hard to tell what effect it'll have on the rest of the UI, so I think it'd be better to remove it after WP 5.3.

Reply:
Sounds good.

Reply:
Thanks for reviewing, I'll put together a PR to remove that property but maybe keep it as draft so that it doesn't get merged just yet.

Reply:
Thanks!